README.md

```diff
@@ -76,6 +76,10 @@ Requirements: User must be authorized on Google Cloud to access the resources-ca
 1. Click mobility-marketplace
 1. Click button for action needed: Upload Files, Delete or Edit on a particular file
 
+## Analytics
```

Comment:
@thekaveman Just an FYI - I'm putting this note here in the README so new people coming into this project know how to get analytics access and know where to look.